pkg/build/build.go

```diff
@@ -678,6 +683,19 @@ func (b *Build) populateWorkspace(ctx context.Context, src fs.FS) error {
 		return err
 	}
 
+	// Write out build settings into workspacedir
+	// For now, just the gcc spec file and just link settings.
+	// In the future can control debug symbol generation, march/mtune, etc.
+	specFile, err := os.Create(filepath.Join(b.WorkspaceDir, ".melange.gcc.spec"))
+	if err != nil {
+		return err
+	}
+	specTemplate := template.New("gccSpecFile")
+	err = template.Must(specTemplate.Parse(gccLinkTemplate)).Execute(specFile, b)
+	if err != nil {
```

Comment:
```suggestion
	if err := template.Must(specTemplate.Parse(gccLinkTemplate)).Execute(specFile, b); err != nil {
```

Reply:
We could hoist this out into an `init()` func but this is such a small template to parse it doesn't really matter.

Reply:
this template is parsed and executed exactly once; irrespective of where/when it happens =)